### DISCUSSION TITLE
[FeatureRequest] Unable to select decimals points for % in the new PIE chart.

### DISCUSSION THREAD

**Discussion by N3v3R3nD** [grafana/grafana#38294] [Dashboards, Panels & Visualizations]
**What happened**:
Unable to select decimals for % in the new PIE chart.

**What you expected to happen**:
Expected there to be decimals after selecting x numbers of decimals in field options. 

**How to reproduce it (as minimally and precisely as possible)**:
Create a PIE chart and try to set decimals of %.

**Environment**:
- Grafana version: 8.0.0
- Data source type & version: InfluxDB 2.0.6
- OS Grafana is installed on: Debian
- User OS & Browser:
- Grafana plugins: 
- Others:

**Comment by zuchka** [comment]
Hi @N3v3R3nD

Is this the behavior you are describing? 

https://play.grafana.org/d/gghjnr6Gz/pie-chart-percent-decimal-test?orgId=1

I can see what you mean: the standard options apply to the panels `value` but not `percent`

**Comment by N3v3R3nD** [comment]
Hello @zuchka 

Yes, indeed that's exactly what I meant, it used to work for both values in the old pie chart.

**Comment by N3v3R3nD** [comment]
Any update on this?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by hjmwijnen** [comment]
I'm facing this same issue.. the percentage in the Pie chart label is rounded to the nearest whole number. This doesn't make sense for some graphs where the expected value (availability, SLA, etc) is somewhere between 99 and 100 percent. Would it be possible to get this configurable?

**Comment by N3v3R3nD** [comment]
Any news on this one?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
